pkg/testsuite/mock/wallet_transactions.go

```diff
@@ -549,20 +549,12 @@ func (w *Wallet) ClaimDelegatorRewards(transactionName string, inputName string)
 
 // Computes the Potential Mana that the output generates until the current slot.
 func (w *Wallet) PotentialMana(api iotago.API, input *utxoledger.Output) iotago.Mana {
-	minDeposit := lo.PanicOnErr(api.StorageScoreStructure().MinDeposit(input.Output()))
-
-	if minDeposit > input.BaseTokenAmount() {
-		return 0
-	}
-
-	excessBaseTokens := input.BaseTokenAmount() - minDeposit
-
-	return lo.PanicOnErr(api.ManaDecayProvider().ManaGenerationWithDecay(excessBaseTokens, input.SlotCreated(), w.currentSlot))
+	return lo.PanicOnErr(iotago.PotentialMana(api.ManaDecayProvider(), api.StorageScoreStructure(), input.Output(), input.SlotCreated(), w.currentSlot))
```

Comment:
I think the whole wallet is moved to iota.go in this PR: https://github.com/iotaledger/iota-core/pull/585
Maybe you need to fix it there?

Reply:
The wallet itself still exists on develop and it doesn't seem to exist in iota.go. I think only the keymanager was moved or something like that?